docs/sources/features/datasources/loki.md

```diff
@@ -111,11 +124,16 @@ log message you're interested in.
 
 ## Templating
 
-Template variables are not yet supported by Loki.
+Instead of hard-coding things like server, application and sensor name in you metric queries you can use variables in their place. Variables are shown as dropdown select boxes at the top of the dashboard. These dropdowns makes it easy to change the data being displayed in your dashboard.
+
+Checkout the [Templating](/reference/templating) documentation for an introduction to the templating feature and the different types of template variables.
 
 ## Annotations
 
-Annotations are not yet supported by Loki.
+You can use any non metric Loki query as a source of annotations. Log content will be used annotation text so
+there is no need for additional mapping.
+
+> Note: Annotations for Loki are only available in Grafana v6.4+
```

Comment:
Cargo culted the note here but are they valuable? We have version switcher in the docs site so I would assume this should be just current docs and not having them separately versioned as for older version these things just won't show.

Reply:
I wonder if we should limit the display of annotation text somehow, so it's not going to blow up a tooltip.

Reply:
> I wonder if we should limit the display of annotation text somehow, so it's not going to blow up a tooltip.

Well I agree with that, but wanted some feedback to the note with grafana version here and whether to have it in docs :D

Reply:
+1 on cargo culting in this case :p